.env.testing

```
@@ -27,3 +27,11 @@ BCRYPT_ROUNDS=4
 LOG_CHANNEL=stack
 LOG_LEVEL=debug
 
+# — a sent invoice in your test database
+SEED_INVOICE_EXTERNAL_ID
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The SEED_INVOICE_EXTERNAL_ID key should be with a value or have an equal sign

(KeyWithoutValue)